pkg/volume/flexvolume/api/objects.go

```diff
@@ -0,0 +1,72 @@
+package api
+
+import (
+	"k8s.io/kubernetes/pkg/api"
+)
+
+// Probe plugin
+
+type FlexVolumeDriverCapabilities struct {
+	AccessMode          api.PersistentVolumeAccessMode
+	DynamicProvisioning bool
+	Attachment          bool
+	LocalAttachment     bool // When Attachement is true, issue the attach/detach calls from the target host, not the controller.
+	CustomMount         bool // This driver has a custom mount/unmount logic. Mount/Unmount must be called.
```

Comment:
Proposal says: 
If Attachment is false, driver will implement mount/unmount.
If Attachment is true, mount will no be called at all in the driver.

What is the case when this CustomMount field is even needed ? Could you give an example ?